Gems/Atom/Feature/Common/Assets/Shaders/PostProcessing/ChromaticAberration.azsl

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#include <scenesrg.srgi>
+
+ShaderResourceGroup PassSrg : SRG_PerPass
+{
+    Texture2D<float4> m_inputColor;
+    RWTexture2D<float4> m_outputColor;
+
+    // Must match the struct in .cpp
```

Comment:
Specify cpp

Reply:
I think it is clear that it is the ChromaticAberrationPass.cpp (there is supposed to be a corresponding XPass.cpp for each X.azsl) and this comment is found elsewhere. I would worry that being more specific could result in copy paste issues in future passes (I know that's what code review is for by it could be missed). Given that, do you still think it should be specified here?

Reply:
Upon reflection, I think this change should be made. The change will be in the next commit.